braidpool_spec.md

```diff
@@ -225,6 +352,15 @@ parent conveys no useful information, since ancestors of each parent are already
 implied when ordering the DAG and including transactions. Visually this means
 that a braid will never have triangles or some other higher order structures.
 
+[kp]
+
+> beads must not name as parents other beads which are
+> ancestors of another parent.
+
+This is pretty standard in DAG based broadcast and consensus algorithms.
+
+[/kp]
+
```

Comment:
Let me introduce you to Iota, which violated this, and was top of my mind years ago...